<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskManagerWidget</class>
 <widget class="QWidget" name="TaskManagerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Менеджер задач</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QPushButton" name="createTaskPushButton">
     <property name="text">
      <string>Создать</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Creator">
      <attribute name="title">
       <string>Создание</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="2">
        <widget class="QGroupBox" name="finishDateTimeGroupBox">
         <property name="title">
          <string>Дата и время завершения</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="1">
           <widget class="QDateTimeEdit" name="finishDateTimeEdit"/>
          </item>
          <item row="0" column="4">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="chooseDateTimePushButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QGroupBox" name="taskInformationGroupBox">
         <property name="title">
          <string>Информация о задаче</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QGroupBox" name="taskNameGroupBox">
            <property name="title">
             <string>Имя задачи</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QLineEdit" name="taskNameEdit"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QGroupBox" name="descriptionGroupBox">
            <property name="title">
             <string>Описание задачи</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QTextEdit" name="descriptionEdit"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QGroupBox" name="periodicityGroupBox">
            <property name="title">
             <string>Периодичность задачи</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QRadioButton" name="onceperiodicityRadioButton">
               <property name="text">
                <string>Однократно</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="dailyPeriodicityRadioButton">
               <property name="text">
                <string>Ежедневно</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="weeklyPeriodicityRadioButton">
               <property name="text">
                <string>Еженедельно</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="nonthlyPeriodicityRadioButton">
               <property name="text">
                <string>Ежемесячно</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="annualyPeriodicityRadioButton">
               <property name="text">
                <string>Ежегодно</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupNameGroupBox">
            <property name="title">
             <string>Группа задачи</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLineEdit" name="groupNameLineEdit"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="startDateTimeGroupBox">
         <property name="title">
          <string>Дата и время начала</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="1">
           <widget class="QDateTimeEdit" name="startDateTimeEdit"/>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="chooseStartDateTimePushButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Editor">
      <attribute name="title">
       <string>Редактирование</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <widget class="QTableWidget" name="editorTableWidget">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>№</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Название</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Дата и время начала</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Дата и время окончания</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Viewer">
      <attribute name="title">
       <string>Просмотр</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <widget class="QTableView" name="viewerTableView">
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
